app/Http/Controllers/Api/V1/AliasController.php

```diff
@@ -0,0 +1,466 @@
+     *             @OA\Items(
+     *                @OA\Property(property="id", type="integer", example="123"),
+     *                @OA\Property(property="name", type="string", example="something"),
+     *             )
+     *          )
+     *       )
+     *    )
+     * )
+     */
+    public function index(Request $request): JsonResponse
+    {
+        $input = $request->all();
+        $jwtUser = array_key_exists('jwt_user', $input) ? $input['jwt_user'] : [];
+
+        try {
+            $aliases = Alias::paginate(Config::get('constants.per_page'), ['*'], 'page');
+
+            Auditor::log([
+                'user_id' => (int)$jwtUser['id'],
+                'action_type' => 'INDEX',
+                'action_name' => class_basename($this) . '@'.__FUNCTION__,
+                'description' => "Alias index",
+            ]);
+
+            return response()->json(
+                $aliases
+            );
+        } catch (Exception $e) {
+            Auditor::log([
+                'user_id' => (int)$jwtUser['id'],
+                'action_type' => 'EXCEPTION',
+                'action_name' => class_basename($this) . '@'.__FUNCTION__,
+                'description' => $e->getMessage(),
+            ]);
+
+            throw new Exception($e->getMessage());
+        }
+    }
+
+    /**
+     * @OA\Get(
+     *      path="/api/v1/aliases/{id}",
+     *      summary="Return a single alias",
+     *      description="Return a single alias",
+     *      tags={"Alias"},
+     *      operationId="AliasController@show",
+     *      security={{"bearerAuth":{}}},
+     *      @OA\Parameter(
+     *         name="id",
+     *         in="path",
+     *         description="alias id",
+     *         required=true,
+     *         example="1",
+     *         @OA\Schema(
+     *            type="integer",
+     *            description="alias id",
+     *         ),
+     *      ),
+     *      @OA\Response(
+     *          response=200,
+     *          description="Success",
+     *          @OA\JsonContent(
+     *              @OA\Property(property="message", type="string"),
+     *              @OA\Property(property="data", type="object",
+     *                  @OA\Property(property="id", type="integer", example="123"),
+     *                  @OA\Property(property="alias", type="string", example="something")
+     *              )
+     *          ),
+     *      ),
+     *      @OA\Response(
+     *          response=404,
+     *          description="Not found response",
+     *          @OA\JsonContent(
+     *              @OA\Property(property="message", type="string", example="not found"),
+     *          )
+     *      )
+     * )
+     */
+    public function show(GetAlias $request, int $id): JsonResponse
+    {
+        $input = $request->all();
+        $jwtUser = array_key_exists('jwt_user', $input) ? $input['jwt_user'] : [];
+
+        try {
+            $alias = Alias::find($id);
```

Comment:
findOrFail?

Reply:
I checked already in the request, I don't need it in this case